create.js

```diff
@@ -46,42 +46,48 @@ module.exports = function create (path, opts, keys) {
   //   - db.getLatest
 
   db.get = function (key, cb) {
-    let isPrivate = false
-    let unbox
+    let doUnbox = false
+    let readKey
     let meta = false
     if (typeof key === 'object') {
-      isPrivate = key.private === true
-      unbox = key.unbox
+      doUnbox = key.private === true
+      readKey = key.unbox
       meta = key.meta
       key = key.id
     }
 
-    if (ref.isMsg(key)) {
-      return db.keys.get(key, function (err, data) {
-        if (err) return cb(err)
-
-        if (!isPrivate) {
-          if (meta) cb(null, { key, value: u.originalValue(data.value), timestamp: data.timestamp })
-          else cb(null, u.originalValue(data.value))
-        }
-        else {
-          const result = db._unbox(data, unbox)
```

Comment:
this call to `_unbox` is un-needed in the case that the message was already auto-unboxed (by the chainmap that `db.keys` has in place).
we only need to call `_db.unbox` when we've been given the `readKey` for a message we couldn't previously decrypt.

Probably doesn't save that many cycles, but seemed silly to run it through there